<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1440</width>
    <height>1024</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QFrame" name="Background">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1440</width>
     <height>1024</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">[objectName^=&quot;Background&quot;]
         {
         background-image: url(:/bg/signup.png);
         }

QLineEdit
{
	background-color:none;
	border:none;
	font-size:20px
}

QCheckBox::indicator
{
	width: 30px;
	height: 30px;
}

QCheckBox::indicator:unchecked
{
 image: url(:/eyes/visibility_off.png);
}

QCheckBox::indicator:checked
{
 image: url(:/eyes/visibility.png);
}

[objectName^=&quot;terms&quot;]::indicator:unchecked
{
 image: none;
}

[objectName^=&quot;terms&quot;]::indicator:checked
{
 image: url(:/others/check.png);
}</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLineEdit" name="name_f">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>540</y>
      <width>231</width>
      <height>51</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Enter Name Here</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="email_f">
    <property name="geometry">
     <rect>
      <x>1070</x>
      <y>540</y>
      <width>231</width>
      <height>51</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Enter Email Here</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="pass1_f">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>700</y>
      <width>201</width>
      <height>51</height>
     </rect>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
    <property name="placeholderText">
     <string>Password</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="pass2_f">
    <property name="geometry">
     <rect>
      <x>1070</x>
      <y>700</y>
      <width>201</width>
      <height>51</height>
     </rect>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
    <property name="placeholderText">
     <string>Repeat Password</string>
    </property>
   </widget>
   <widget class="QPushButton" name="signup_b">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>890</y>
      <width>271</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
background-color: none;
border:none;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="login_b">
    <property name="geometry">
     <rect>
      <x>1050</x>
      <y>890</y>
      <width>271</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
background-color: none;
border:none;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QCheckBox" name="p1_cb">
    <property name="geometry">
     <rect>
      <x>960</x>
      <y>710</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QCheckBox" name="p2_cb">
    <property name="geometry">
     <rect>
      <x>1280</x>
      <y>710</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QCheckBox" name="terms">
    <property name="geometry">
     <rect>
      <x>747</x>
      <y>810</y>
      <width>61</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="../Resources/Signup/signupres.qrc"/>
  <include location="../Resources/universal.qrc"/>
 </resources>
 <connections/>
</ui>
